scripts/Engineering/gui/engineering_diffraction/tabs/fitting/plotting/plot_view.py

```diff
@@ -148,7 +148,7 @@ def read_fitprop_from_browser(self):
     def update_browser(self, status, func_str, setup_name):
         self.fit_browser.fitResultsChanged.emit(status)
```

Comment:
This is called after each iteration of the sequential fit and updates the status displayed in the browser. So at the end of a sequential fit it should display the status of the last fit. I did wonder whether to just update the status after the last fit only - I can do that in this PR by separating the updating of the status from the loading/saving function as setup.

Reply:
Thanks - that explains it. I don't think it matters whether you call fitResultsChanged more or less frequently. The point is that the window title isn't updated in sync with this so the title can say "success" while the status says something which is not success.

Have a look in FitPropertyBrowser::finishHandle. This emits both fitResultsChanged and changeWindowTitle. So maybe you need to emit changeWindowTitle also in the engineering code?

Reply:
Ah I understand now, I will try emitting the `changeWindowTitle` - thanks for the suggestion!